include/maliput/plugin/create_road_network.h

```diff
@@ -48,5 +49,13 @@ namespace plugin {
 std::unique_ptr<maliput::api::RoadNetwork> CreateRoadNetwork(const std::string& road_network_loader_id,
                                                              const std::map<std::string, std::string>& properties);
 
+/// Creates a maliput::plugin::RoadNetworkLoader using the specified plugin id.
+/// @param road_network_loader_id RoadNetworkLoader plugin id to be used.
+/// @returns A maliput::plugin::RoadNetworkLoader.
+///
+/// @throws maliput::common::assertion_error When `road_network_loader_id` is not found.
+/// @throws maliput::common::assertion_error When the plugin isn't a RoadNetworkLoader plugin type.
+std::unique_ptr<maliput::plugin::RoadNetworkLoader> CreateRoadNetworkLoader(const std::string& road_network_loader_id);
```

Comment:
Does it make sense to call it `MakeRoadNetworkLoader`?

Reply:
Done